Launcher/Program.cs

```diff
@@ -94,13 +94,16 @@ static void Main(string[] args)
 
             if (environment.EndsWith("-desktop"))
             {
-                var info = new ProcessStartInfo
-                {
-                    UseShellExecute = false,
-                    FileName  = Config.Get("desktop-exe"),
-                    Arguments = Config.Get("desktop-http-port")
-                };
-                Process.Start(info);
+                // For the web version of the Lean Desktop Client launch the webpage rather than the application
+                var httpProtocol = Config.Get("desktop-http-protocol", "http");
+                var httpHost = Config.Get("desktop-http-host", "localhost");
+                var httpPort = Config.Get("desktop-http-port", "1234");
+                var url = string.Format("{0}://{1}:{2}", httpProtocol, httpHost, httpPort);
+
+                // Once the webserver is in place launch it here first
+
+                // Open the default webpage for the Lean Desktop website
+                Process.Start(url);
```

Comment:
Cool! Didn't know that this would be relayed as a shell execute.I guess since it's a well-formed URL, the OS knows exactly what to use to open the path.